### PR TITLE
Cbrune/andrews pr branch for integration tests for tax num fix

### PR DESCRIPTION
**Pull request by cbrunefearless**
### Change Details
Fix integration tests for tax num fix.  Looks like the mapping for v2 tax number was wrong from what I was originally told.  Was originally told to map it to extension and than it changed to care team.  

### Acceptance Validation
All the IT test pass and everything with tax number is correct.

### Feedback Requested
Look at the code and determine any inefficiencies.

### External References

- [BFD-xxx](https://jira.cms.gov/browse/BFD-xxx)

### Security Implications

- [ ] new software dependencies

- [ ] altered security controls

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

### CODE REVIEW

File: apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/utils/EndpointJsonResponseComparatorIT.java [hunk 1]
```diff
@@ -164,7 +163,7 @@ public void verifyCorrectEndpointResponse() {
    * Generates the "golden" files, i.e. the approved responses to compare to. Run by commenting out
    * the <code>@Ignore</code> annotation and running this method as JUnit.
    */
-  @Ignore
+  // @Ignore
```
Comment (mackec1) [inline apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/utils/EndpointJsonResponseComparatorIT.java:166]: Don't we leave the @Ignore active after generating the .json files?

Comment (cbrunefearless) [reply apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/utils/EndpointJsonResponseComparatorIT.java:166]: yes thannk u!!

**Review by mackec1 — APPROVED** [review]
Nice job!!!

**Review by jzulim — APPROVED** [review]
Looks nice. Thanks for jumping on this so fast!

**Pull request merged**